mlos_bench/mlos_bench/util.py

```diff
@@ -100,3 +71,25 @@ def check_required_params(config: Dict[str, Any], required_params: Iterable[str]
         raise ValueError(
             "The following parameters must be provided in the configuration"
             + f" or as command line arguments: {missing_params}")
+
+
+def get_git_info(path: str = ".") -> Tuple[str, str]:
```

Comment:
```suggestion
def get_git_info(path: str = __file__) -> Tuple[str, str]:
```

I think otherwise this will fail if it's run with `$PWD` != somewhere in the repo.

Reply:
great point. I'll fix it and add more checks in teh unit test